webknossos/local_wk_setup.sh

```diff
@@ -1,7 +1,7 @@
 function export_vars {
     export WK_TOKEN=1b88db86331a38c21a0b235794b9e459856490d70408bcffb767f64ade0f83d2bdb4c4e181b9a9a30cdece7cb7c65208cc43b6c1bb5987f5ece00d348b1a905502a266f8fc64f0371cd6559393d72e031d0c2d0cabad58cccf957bb258bc86f05b5dc3d4fff3d5e3d9c0389a6027d861a21e78e3222fb6c5b7944520ef21761e
     export WK_URL=http://localhost:9000
-    export DOCKER_TAG=master__20617
+    export DOCKER_TAG=master__10303
```

Comment:
@normanrz Just noticed that this is not a valid tag, I'm updating it now in another PR to [master__21723](https://hub.docker.com/layers/scalableminds/webknossos/master__21723/images/sha256-22d498777f8061fa653a9590c4a56f50a1c5e80ac4b433b60495f351c68534a8?context=explore)